src/index.ts

```diff
@@ -1,2 +1,2 @@
 export { default } from './Pagination';
-export type { PaginationProps } from './interface';
+export type * from './interface';
```

Comment:
@afc163 This change is causing an issue when I am attempting to build with the latest version of `rc-pagination`
I'm seeing:
```
> tsc -b

node_modules/rc-pagination/lib/index.d.ts:2:1 - error TS1383: Only named exports may use 'export type'.

2 export type * from './interface';
  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

If you change this to `export * from `./interface` I believe things will do what you want and avoids this typescript error

Reply:
Could you provide a reproduce codesandbox?